### PR TITLE
Add troubleshooting guide

### PR DESCRIPTION
**Pull request by dplumb94**
Signed-off-by: Darian Plumb <dplumb@bitwise.io>

### CODE REVIEW

File: track_and_trace/docs/troubleshooting_guide.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+AssetTrack demo application.
+
+### AssetTrack Test ###
+
+Finally, we will go through the AssetTrack demo application to ensure that the
+app is functioning properly and communicating with the Grid services.
+
+Complete the following actions:
+
+* Navigate to `localhost:8021` in a web browser.
```
Comment (eloaverona) [inline track_and_trace/docs/troubleshooting_guide.md]: I think @chenette would tell you to make this a ordered list, as they are steps to be performed in a certain order.

File: track_and_trace/docs/troubleshooting_guide.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+## Verify Communication and Functionality ##
+
+The steps in this section cover testing Grid Track and Trace using the
+AssetTrack demo application.
+
+### AssetTrack Test ###
+
+Finally, we will go through the AssetTrack demo application to ensure that the
```
Comment (eloaverona) [inline track_and_trace/docs/troubleshooting_guide.md]: Would it be worth it to include samples of what the logs look like if each of those steps were completed successfully?

Comment (dplumb94) [reply track_and_trace/docs/troubleshooting_guide.md]: I wasn't intending to put that level of detail in this guide. Also any errors should come through the UI.

File: track_and_trace/docs/troubleshooting_guide.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+organization.
+4. Navigate to the View Agents page.
+    * You should see the user you just created in the list.
+5. Navigate to the Profile page.
+    * You should see a public key and organization ID.
+6. Navigate to the Add Asset page, and fill out and submit the form.
+7. Navigate to the View Assets page.
+    * You should see the newly created asset in the list.
+8. Log out and create a new user/organization using the Get Started form.
+9. Log in to the first agent.
```
Comment (jsmitchell) [inline track_and_trace/docs/troubleshooting_guide.md]: s/Log in to/Log in as/

**Pull request merged**